Resources/Prototypes/_LostParadise/Entities/Objects/Consumable/Drinks/drinks.yml

```
@@ -0,0 +1,31 @@
+#- type: entity
```

yamllint

[warning] missing starting space in comment

(comments)